### PR TITLE
Fix release scripts for arm64

### LINKED ISSUE #212: release.yaml: Cross compile libpcap.a for arm64

### PR DESCRIPTION
**Pull request by brb**
Tested locally by running `make release`.

Fix #212

### CODE REVIEW

File: local-release.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    make clean
+	echo "Building release binary for ${OS}/${ARCH}..."
+    make pwru TARGET_GOARCH=${ARCH} LIBPCAP_ARCH=${LIBPCAP_ARCH} CC=${CC} 
+
+	test -d release/${OS}/${ARCH} || mkdir -p release/${OS}/${ARCH}
+	tar -czf release/pwru-${OS}-${ARCH}.tar.gz pwru
+	(cd release && sha256sum pwru-${OS}-${ARCH}.tar.gz > pwru-${OS}-${ARCH}.tar.gz.sha256sum)
+	rm -r release/${OS}
+done
```
Comment (tklauser) [inline local-release.sh:24]: nit: looks like there is a mix of tabs and spaces used for indentation here. Would be nice to consistently use one or the other.

File: Makefile [hunk 2]
```diff
@@ -54,7 +48,6 @@ clean:
 	rm -f kprobemultipwru_bpf*
 	rm -f kprobepwruwithoutoutputskb_bpf*
 	rm -f kprobemultipwruwithoutoutputskb_bpf*
-	rm -rf ./release
 	cd libpcap/ && make clean
```
Comment (jschwinger233) [inline Makefile:51]: There is no `Makefile` under `libpcap/` until we run `./configure` to generate one from `Makefile.in`, and `make clean` will fail with error ` No rule to make target 'clean' ` before that.

File: local-release.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        CC=gcc
+    fi
+
+    make clean
+    echo "Building release binary for ${OS}/${ARCH}..."
+    make pwru TARGET_GOARCH=${ARCH} LIBPCAP_ARCH=${LIBPCAP_ARCH} CC=${CC} 
+
+    test -d release/${OS}/${ARCH} || mkdir -p release/${OS}/${ARCH}
+    tar -czf release/pwru-${OS}-${ARCH}.tar.gz pwru
+    (cd release && sha256sum pwru-${OS}-${ARCH}.tar.gz > pwru-${OS}-${ARCH}.tar.gz.sha256sum)
```
Comment (jschwinger233) [inline local-release.sh:22]: Maybe we can try goreleaser in the future :smiley:

Comment (brb) [reply local-release.sh:22]: As long as it's able to statically link with CGO, I am all in :smile:

**Review by jschwinger233 — APPROVED** [review]

**Pull request merged**